extensions/doc/makeLuaDocs.lua

```diff
@@ -230,8 +231,8 @@ print("++ Parsing manual...")
 -- ignorable, as they're likely links within the manual itself to lists already included in a manual section,
 -- but post them just in case I need to look for them later if the format changes.
 
-    local a, b, k = 1, 0, ""
-    local text, posTable = nil, nil
+    a = 1
```

Comment:
as does this

Reply:
`b` and `k` are already defined as `local` [earlier in the code](https://github.com/cmsj/hammerspoon/blob/d5c32b2799b534ae355cfeb3615831f052cc1f42/extensions/doc/makeLuaDocs.lua#L199). Their initial values are also overwritten [here](https://github.com/cmsj/hammerspoon/blob/d5c32b2799b534ae355cfeb3615831f052cc1f42/extensions/doc/makeLuaDocs.lua#L203).

`text` and `posTable` are also already defined as `local` [earlier in the code](https://github.com/cmsj/hammerspoon/blob/d5c32b2799b534ae355cfeb3615831f052cc1f42/extensions/doc/makeLuaDocs.lua#L200).